core/frontend/src/ViewCreator2d.ts

```diff
@@ -37,7 +37,7 @@ export interface ViewCreator2dOptions {
  * const viewCreator = new ViewCreator2d(imodel);
  * const models = await imodel.models.queryProps({ from: "BisCore.GeometricModel2d" });
  * if (models.length > 0)
- * const view = await viewCreator.createViewForModel(models[0].id!, models[0].classFullName);
+ * const view = await viewCreator.createViewForModel(models[0].id!);

```

Comment:
Why throw away the classFullName you just queried to make `createViewForModel` query for it again?

Reply:
This relates to this thread: https://github.com/imodeljs/imodeljs/discussions/323#discussioncomment-295770

Reply:
> Querying the model type from the backend is what I was thinking. I suppose you could have an optional modelType argument for cases where the caller already knows the type. If so, it would be preferable to use an enum or a union of string literals instead of accepting an arbitrary (and verbose) string - there are only 3 valid inputs.

You kept the optional argument, which exists for cases in which callers already know the model type. This caller already knows the model type.